packages/elements/src/components/ino-snackbar/ino-snackbar.scss

```diff
@@ -58,3 +60,11 @@ ino-snackbar {
     }
   }
 }
+
+@each $type in $snackbar-types {
+  ino-snackbar[ino-type="#{$type}"] {
```

Comment:
We should provide CSS vars to customize the color of these different types

Reply:
✅